docs/products/cassandra/get-started.md

```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
+    project                 = var.aiven_project_name
+    cloud_name              = "google-europe-west1"
+    plan                    = "startup-4"
+    service_name            = "cassandra-sample"
+    maintenance_window_dow  = "sunday"
+    maintenance_window_time = "22:00:00"
+    termination_protection  = true
+
+    cassandra_user_config {
+        migrate_sstableloader = true
+        service_log = true
+    }
+    }
+
+    output "cassandra_service_host" {
+    value = aiven_cassandra.example_cassandra.service_host
+    }
+
+    output "cassandra_service_port" {
+    value = aiven_cassandra.example_cassandra.service_port
+    }
+
+    output "cassandra_service_username" {
+    value = aiven_cassandra.example_cassandra.service_username
+    }
+
+    output "cassandra_service_password" {
+    value     = aiven_cassandra.example_cassandra.service_password
+    sensitive = true
+    }
+    ```
+
+1. Run `terraform plan` > `terraform apply --auto-approve`.
+
+</TabItem>
+<TabItem value="3" label="K8s">
+1. Update file `cassandra-sample.yaml`:
+
+   - Add `service_log: true` and `terminationProtection: true`.
+   - Update `maintenanceWindowDow: sunday` and `maintenanceWindowTime: 22:00:00`.
+
+   ```yaml
+   apiVersion: aiven.io/v1alpha1
+   kind: Cassandra
+   metadata:
+   name: cassandra-sample
+   spec:
+   authSecretRef:
+       name: aiven-token
+       key: token
+
+   connInfoSecretTarget:
+       name: cassandra-secret
+
+   userConfig:
+       migrate_sstableloader: true
+       service_log: true
+
+   project: PROJECT_NAME
+   cloudName: google-europe-west1
+   plan: startup-4
+
+   maintenanceWindowDow: sunday
+   maintenanceWindowTime: 22:00:00
+   terminationProtection: true
+   ```
+
+1. Update the service by applying the configuration:
+
+   ```go
+   kubectl apply -f cassandra-sample.yaml
+   ```
+
+1. Review the resource you updated with the following command:
+
+   ```go
+   kubectl describe cassandra.aiven.io cassandra-sample
+   ```
+
+The resource can stay in the `REBUILDING` state for a couple of minutes. Once the state
+changes to `RUNNING`, you are ready to access it.
+
+See the available configuration options in
+[Aiven Operator for Kubernetes®: Cassandra](https://aiven.github.io/aiven-operator/api-reference/cassandra.html).
+</TabItem>
+</Tabs>
+
+## Connect to the service{#connect-to-service}
+
+<Tabs groupId="group1">
+<TabItem value="1" label="Console" default>
+1. Log in to the [Aiven Console](https://console.aiven.io/), and go to your
+   organization > project > Aiven for Apache Cassandra service.
+1. On the <ConsoleLabel name="overview"/> page of your service, click
+   **Quick connect**.
+1. In the **Connect** window, select a tool or language to connect to your service, follow
+   the connection instructions, and click **Done**.
+
+   ```bash
+   # Sample command for connecting with the cqlsh client
+
+   ./cqlsh --ssl -u USER_NAME -p PASSWORD HOST_NAME PORT_NUMBER
```

Comment:
this command should be the same both for Console and Terraform? this version is missing the host and port, so most likely won't work for the customer?

Reply:
In this command here, I've included placeholders (`HOST_NAME` and `PORT_NUMBER`) so that the user can replace them with the actual values. Same for `USER_NAME` and `PASSWORD` in this command. In any case, this command is only included as an example coz the actual guidance we're providing here is "Follow the steps 1-3".
In the Terraform flow, the corresponding command differs coz it's using the envir variables we saved specifically for the TF flow purposes (see lines 105-108) so they are not available in the console flow.